src/john.c

```diff
@@ -1947,6 +1948,7 @@ static void john_done(void)
 			/* We already printed to stderr from signals.c */
 			log_event("%s", abort_msg);
 		} else if (children_ok) {
+			log_event("Candidates tried: %"PRIu64"p", status.cands);
```

Comment:
I'd also log suppressor's hits and misses, which is info that is otherwise impossible to access at session end (no way to press `s` at exactly the right moment).

Reply:
> That would be a new function in status.c, right?

No, it's `status.suppressor_hit` and `status.suppressor_miss` just like `status.cands` that you use.